base/markdown/parse/util.jl

```diff
@@ -11,12 +11,13 @@ macro dotimes(n, body)
 end
 
 const whitespace = " \t\r"
+const whitespace8 = map(UInt8,collect(whitespace))
```

Comment:
Instead of doing this, I wonder whether it wouldn't be more logical to do `Char(peek(io)) in whitespace`. Converting to `UInt8` sounds backwards. There's also the very similar `isspace` function.